docs/src/index.md

```diff
@@ -125,6 +125,16 @@ g["mydataset"] = rand(3,5)
 write(g, "mydataset", rand(3,5))
 ```
 
+## Row- and column-major order
```

Comment:
We have overlap with this section https://juliaio.github.io/HDF5.jl/stable/#Details   which probably needs to be renamed.

Reply:
Ah I see! I was not even aware of the exact thing I wanted, because it's at the bottom of the doc with a irrelevant name! It make sense to just rename it.